ocamltest/ocaml_variables.ml

```diff
@@ -50,6 +50,12 @@ let cc = make ("cc",
 let cflags = make ("cflags",
   "Flags passed to the C compiler")
 
+let cxx = make ("cxx",
+  "Command to use to invoke the C++ compiler")
+
+let ccomp_type = make ("ccomp_type",
+  "Type of C compiler (msvc, cc, etc.)")
```

Comment:
```suggestion
  "Type of C compiler (["cl"] for MSVC or ["cc"])")
```

There are predicates testing for MSVC, which makes exposing this as a variable seem a bit strange - but I'm not sure there's a clearer way to communicate it to the shell script?